src/README.md

```diff
@@ -239,5 +239,29 @@ juju config trove nova-keypair=trove-keypair
 ```
 
 
+## Troubleshooting
+
+Each database instance deployed has **2 ports** for the:
+- `tenant-net`
+- `trove-net`
+
+The **tenant-net port** has a security group (`trove_sg-uuid`) that allows the RabbitMQ traffic.
+
+The **trove-net port** has a security group (`trove-sec-group`) that allows the egress traffic 
+through the `mgmt-net`.
```

Comment:
Also, this security group is created and used if the charm `management-security-groups` config option is not set. If it is set, it'll have whatever rules are set in that group. This will have to be corrected.

Reply:
Updated it